packages/patternfly-4/react-core/scripts/copyStyles.js

```diff
@@ -41,3 +44,20 @@ copySync(join(pfDir, 'assets/fonts'), join(stylesDir, 'assets/fonts'), {
   }
 });
 writeFileSync(join(stylesDir, 'base.css'), stringifyCSS(ast));
+
+// Copy component css
+const srcDir = join('./src');
+const distDirs = [join('./dist/js'), join('./dist/esm'), join('./dist/umd')];
+
+const files = glob.sync('**/*.css', {
+  cwd: srcDir
+});
+
+files.forEach(file => {
```

Comment:
I'm concerned about adding any support for inline/required css directly to react-core.  Maybe we can consider the use of CSS directly an antipattern and make sure we don't make it easy for people to do.  For now, can we put this CSS directly into react-styles instead so we don't add the ability for direct inclusion through react-core?

Reply:
ok, will try changing this to something more like:
```
import '@patternfly/react-styles/css/components/Tooltip/popover-base-styles.css';
```
as a next step...

Reply:
updated! PTAL @dgutride